source/_integrations/home_connect.markdown

```diff
@@ -103,7 +103,50 @@ The integration configuration will ask for the *Client ID* and *Client Secret* c
 ## Actions
 
 The Home Connect integration makes various actions available.
-Available actions: `set_option_active`, `set_option_selected`, `pause_program`, `resume_program`, `select_program`, `start_program` and `change_setting`
+Available actions: `set_program_and_options`, `set_option_active`, `set_option_selected`, `pause_program`, `resume_program`, `select_program`, `start_program` and `change_setting`
+
+### Action `home_connect.set_program_and_options`
+
+Starts or selects a program. If the `program` attribute is not set, this action sets the options for the active or the selected program.
+
+| Data attribute    | Optional | Description                                      |
+|---------------------------|----------|--------------------------------------------------|
+| `device_id` | no | ID of the device. |
+| `affects_to` | no | Selects if the program affected by the action should be the active or the selected program. |
+| `program` | yes | Program to select. If set, it will start or select a program depending on `affects_to`. |
+| `consumer_products_cleaning_robot_option_reference_map_id` | yes | Defines which reference map is to be used. |
+| `consumer_products_cleaning_robot_option_cleaning_mode` | yes | Defines the favoured cleaning mode. |
+| `consumer_products_coffee_maker_option_bean_amount` | yes | Describes the amount of coffee beans used in a coffee machine program. |
+| `consumer_products_coffee_maker_option_fill_quantity` | yes | Describes the amount of water (in ml) used in a coffee machine program. |
+| `consumer_products_coffee_maker_option_coffee_temperature` | yes | Describes the coffee temperature used in a coffee machine program. |
+| `consumer_products_coffee_maker_option_bean_container` | yes | Defines the preferred bean container. |
+| `consumer_products_coffee_maker_option_flow_rate` | yes | Defines the water-coffee contact time. The duration extends to coffee intensity. |
+| `consumer_products_coffee_maker_option_multiple_beverages` | yes | Defines if double dispensing is enabled. |
+| `consumer_products_coffee_maker_option_coffee_milk_ratio` | yes | Defines the amount of milk. |
+| `consumer_products_coffee_maker_option_hot_water_temperature` | yes | Defines the temperature suitable for the type of tea. |
+| `b_s_h_common_option_start_in_relative` | yes | Defines in how many time the program should start. |
+| `dishcare_dishwasher_option_intensiv_zone` | yes | Defines if the cleaning is done with higher spray pressure on the lower basket for very dirty pots and pans. |
+| `dishcare_dishwasher_option_brilliance_dry` | yes | Defines if the program sequence is optimized with a special drying cycle to ensure more shine on glasses and plastic items. |
+| `dishcare_dishwasher_option_vario_speed_plus` | yes | Defines if the program run time is reduced by up to 66% with the usual optimum cleaning and drying. |
```

Comment:
```suggestion
| `dishcare_dishwasher_option_vario_speed_plus` | yes | Defines if the program duration is shortened dynamically (up to 66% less run time) with the usual optimum cleaning and drying. |
```
Found it! How about we say it like this?
![image](https://github.com/user-attachments/assets/ae2b7926-f2e0-493e-bbe7-74b7a6e0b7dc)